dev-vcs/tig/tig-2.2.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="A text mode interface for git"
+DESCRIPTION="
+Tig functions mainly as a git repository browser. It can also help in the \
+staging changes for commit at chunk level and act as a pager for output \
+different git commands."
+HOMEPAGE="http://jonas.nitro.dk/tig/"
+SOURCE_URI="http://jonas.nitro.dk/tig/releases/tig-2.2.tar.gz"
+CHECKSUM_SHA256="8f5213d3abb45ca9a79810b8d2a2a12d941112bc4682bcfa91f34db74942754c"
+REVISION="1"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2006-2016 Jonas Fonseca"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	tig$secondaryArchSuffix = $portVersion
+	cmd:tig$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	cmd:git
+	"
+BUILD_REQUIRES="
+	devel:libiconv$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libreadline$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:asciidoc
+	cmd:autoconf
+	cmd:find
+	cmd:git
+	cmd:libtoolize
```

Comment:
**`cmd:libtoolize`** should be followed by **`$secondaryArchSuffix`** like **`cmd:gcc`** and **`cmd:ld`** are.